Amazon.QLDB.Driver/driver/QldbDriver.cs

```diff
@@ -33,15 +35,18 @@ public class QldbDriver : IQldbDriver
                 "SELECT VALUE name FROM information_schema.user_tables WHERE status = 'ACTIVE'";
 
         private readonly SessionPool sessionPool;
+        private readonly ILogger logger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="QldbDriver"/> class.
         /// </summary>
         ///
         /// <param name="sessionPool">The ledger to create sessions to.</param>
-        internal QldbDriver(SessionPool sessionPool)
+        /// <param name="logger">The logger of the class.</param>
```

Comment:
> Logger to be used by this
to be consistent with other logger docstring

Reply:
fixed.